tests_pytest/pytorch/gptq/test_gradual_act_quantization.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2024 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from unittest.mock import Mock
+
+import pytest
+import torch
+
+from model_compression_toolkit.core.pytorch.pytorch_device_config import get_working_device
+from model_compression_toolkit.trainable_infrastructure.pytorch.annealing_schedulers import LinearAnnealingScheduler
+from model_compression_toolkit.gptq import GradientPTQConfig, GradualActivationQuantizationConfig, LinearAnnealingConfig
+from model_compression_toolkit.gptq.pytorch.quantizer.gradual_activation_quantization import (
+    GradualActivationQuantizerWrapper, get_gradual_activation_quantizer_wrapper_factory)
+
+
+@pytest.fixture
+def x():
```

Comment:
can you briefly describe this function, what is it doing, and what the decorator is for?
add similar description in other test functions that use pytest features or follow common practice.
this is only for educational purposes - when other members will start to write test in pytest and convert our tests to this package, I want them to follow these guidelines.

Reply:
Yes I meant in the code, but on second thought, after your session it is better that if someone would encounter this (or other pytest features) in your tests they just ask you or search the web for answers instead of swamping our code with comments